server/server.js

```diff
@@ -110,6 +124,7 @@ APP.post("/pet", async (req, res) => {
   res.send(newpet);
 });
 
+// endpoint to edit pet
```

Comment:
This comment doesn't add much. I grok this from `put("pet`.  I'd leave it off or add more. [controversial opinion]

Reply:
I think I'd like to keep this one, purely to keep all the endpoints consistent.